stdlib/public/runtime/Float16Support.cpp

```diff
@@ -161,4 +164,10 @@ SWIFT_RUNTIME_EXPORT unsigned short __truncdfhf2(double d) {
   return __gnu_f2h_ieee(f);
```

Comment:
Does `__gnu_f2h_ieee` exist, or does it have another name in AEBI?

Reply:
It is spelt `__aeabi_f2h`.  The `__gnu_f2h_ieee` is a GNU RT specific symbol IIRC (should only be used for libgcc not when building with `-rtlib=compiler-rt`).